### PR TITLE
Feature/eui 2292 judicial header 5

### PR DESCRIPTION
**Pull request by ficklephil**
Judicial Header.

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: yarn.lock [hunk 1]
```diff
@@ -1569,6 +1577,14 @@ amdefine@>=0.0.4:
   resolved "https://registry.yarnpkg.com/amdefine/-/amdefine-1.0.1.tgz#4a5282ac164729e93619bcfd3ad151f817ce91f5"
   integrity sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=
 
+angular-tree-component@^8.5.6:
```
Comment (adnanakgun) [inline yarn.lock:1580]: Has the yarn.lock file been deleted and rebuild in this PR? Why are there so many changes to the lock file?

**Review by adnanakgun — APPROVED** [review]

File: src/app/app.constants.ts [hunk 2]
```diff
@@ -30,29 +31,109 @@ const FooterDataNavigation = {
...
+    roles: [
+      'caseworker-sscs-judge',
+      'caseworker-sscs-panelmember',
+      'caseworker-cmc-judge',
+      'caseworker-divorce-judge',
+      'caseworker-divorce-financialremedy-judiciary',
+      'caseworker-probate-judge',
+      'caseworker-ia-iacjudge',
+      'caseworker-publiclaw-judiciary',
+    ],
```
Comment (rayliangatos) [inline src/app/app.constants.ts:45]: roles hard coded? possible to move to LD or config?

Comment (ficklephil) [reply src/app/app.constants.ts:45]: It is in the LD Config, this is the fall over, just in case LD stops working.

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**